tests/common/test_logger.py

```diff
@@ -207,8 +207,11 @@ def test_logger_should_log_in_utc(self):
 
         with open(file_path, "r") as log_file:
             log = log_file.read()
-            time_in_file = datetime.strptime(log.split(LogLevel.STRINGS[logger.LogLevel.INFO])[0].strip()
-                                             , u'%Y/%m/%d %H:%M:%S.%f')
+            try:
+                time_in_file = datetime.strptime(log.split(LogLevel.STRINGS[logger.LogLevel.INFO])[0].strip()
```

Comment:
datetime.isoformat()

Reply:
sorry, not in the test, on line 92 logger.py

Reply:
Nice I didnt know something like this existed! 1 problem with this though, it doens't print the 'Z' in the end

```python
datetime.isoformat(datetime.utcnow())
'2019-11-21T19:24:40.652963'